pkg/cloud/amazon/eks/eks.go

```diff
@@ -13,34 +17,30 @@ import (
 	"github.com/pkg/errors"
 )
 
-// EksClusterOptions contains some functions to interact with EKS
-type EksClusterOptions struct {
+// eksAPIHandler contains some functions to interact with and serves as an abstraction of the EKS API
```

Comment:
"eksAPIHandler contains some functions to interact with EKS ..."

Reply:
Interact with, and serve as an abstraction of EKS, both phrases are supposed to lead to EKS

Reply:
Yeah, I don't think this is grammatically correct though.